FWCore/FWLite/interface/BareRootProductGetterBase.h

```diff
@@ -0,0 +1,119 @@
+#ifndef FWCore_FWLite_BareRootProductGetterBase_h
+#define FWCore_FWLite_BareRootProductGetterBase_h
+// -*- C++ -*-
+//
+// Package:     FWLite
+// Class  :     BareRootProductGetterBase
+//
+/**\class BareRootProductGetterBase BareRootProductGetterBase.h FWCore/FWLite/interface/BareRootProductGetterBase.h
+
+ Description: <one line class summary>
+
+ Usage:
+    <usage>
+
+*/
+//
+// Original Author:  Chris Jones
```

Comment:
How about updating this comment to say the code was moved so the git history doesn't conflict with this comment?